src/components/UNSTABLE__Shell/PopoverAccountItem/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Popover account item.
+ * @copyright IBM Security 2020
+ */
+
+import { func, node, string } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as accountNamespace } from '../PopoverAccount';
+
+const namespace = appendComponentNamespace(accountNamespace, 'account');
+
+function PopoverAccountItem({ children, id, onClick, ...other }) {
+  return (
+    <li className={namespace}>
+      <button
+        className={`${namespace}__button`}
+        onClick={onClick}
+        value={id}
+        {...other}
```

Comment:
Did you try to use a Carbon `Button` here?
(The reason I mention that here + in another popover subcomponent is for the built-in a11y attributes, mainly. They are very useful & I would rather take advantage of those + add style overrides than use a plain `button` element to avoid style overrides -- does that make sense? And do you have concerns?)

Reply:
Resolving thread as per Slack conversation